docs/azure-config.md

```diff
@@ -39,7 +39,7 @@ AZURE_BACKUP_RESOURCE_GROUP=Ark_Backups
 az group create -n $AZURE_BACKUP_RESOURCE_GROUP --location WestUS
 
 # Create the storage account
-AZURE_STORAGE_ACCOUNT_ID="ark`cat /proc/sys/kernel/random/uuid | cut -d '-' -f5`"
+AZURE_STORAGE_ACCOUNT_ID="ark$(uuidgen | cut -d '-' -f5 | tr '[A-Z]' '[a-z]')"
```

Comment:
I checked various docker images to see if they have uuidgen:

- `fedora:27`: ✅ 
- `centos:7`: ✅
- `ubuntu:16.04`: ❌ but if you install `uuid-runtime` then ✅ 
- `alpine:3.7`: ❌ but if you install `util-linux` then ✅ 

We probably should say that we're just trying to come up with a globally unique identifier, 3-24 lowercase alphanumeric characters in length (https://docs.microsoft.com/en-us/azure/architecture/best-practices/naming-conventions). If the `uuidgen` command isn't working for them, they can manually generate one too.

Reply:
Ah, thanks for checking that. Pushed an update to the instructions.